doc/sphinx/changes.rst

```diff
@@ -8,6 +8,991 @@ Recent changes
+    rather than interpreted terms. In particular, their typeclass
+    arguments will not be inferred. In general, the previous behaviour
+    can be emulated by replacing `auto using foo` with `pose proof
+    foo; auto`.
+
+  - Argument order for the Ltac2 combinators `List.fold_left2` and
+    `List.fold_right2` changed to be the same as in OCaml.
+
+  - :cmd:`Import`\ing a module containing a mutable Ltac2 definition
+    does not undo its mutations. Replace `Ltac2 mutable foo :=
+    some_expr.` with `Ltac2 mutable foo := some_expr. Ltac2 Set foo :=
+    some_expr.` to recover the previous behaviour.
+
+  - Some :ref:`renaming <820_renaming_stdlib>` in the standard
+    library. Deprecations are provided for a smooth transition.
+
+See the `Changes in 8.20.0`_ section below for the detailed list of changes,
+including potentially breaking changes marked with **Changed**.
+Coq's `reference manual for 8.20 <https://coq.github.io/doc/v8.20/refman>`_,
+`documentation of the 8.20 standard library <https://coq.github.io/doc/v8.20/stdlib>`_
+and `developer documentation of the 8.20 ML API <https://coq.github.io/doc/v8.20/api>`_
+are also available.
+
+Théo Zimmermann with help from Ali Caglayan and Jason Gross maintained
+`coqbot <https://github.com/coq/bot>`_ used to run Coq's CI and other
+pull request management tasks.
+
+Jason Gross maintained the `bug minimizer <https://github.com/JasonGross/coq-tools>`_
+and its `automatic use through coqbot <https://github.com/coq/coq/wiki/Coqbot-minimize-feature>`_.
+
+Jaime Arias and Erik Martin-Dorel maintained the
+`Coq Docker images <https://hub.docker.com/r/coqorg/coq>`_
+and the Docker Keeper tool used to build and keep those images up to date.
+Cyril Cohen, Vincent Laporte, Pierre Roux and Théo Zimmermann
+maintained the `Nix toolbox <https://github.com/coq-community/coq-nix-toolbox>`_
+used by many Coq projects for continuous integration.
+
+Ali Caglayan, Emilio Jesús Gallego Arias, Rudi Grinberg and
+Rodolphe Lepigre maintained the
+`Dune build system for OCaml and Coq <https://github.com/ocaml/dune/>`_
+used to build Coq itself and many Coq projects.
+
+The opam repository for Coq packages has been maintained by
+Guillaume Claret, Guillaume Melquiond, Karl Palmskog and Enrico Tassi with
+contributions from many users. A list of packages is `available on the Coq website <https://coq.inria.fr/coq-package-index>`_.
+
+Coq 8.20 was made possible thanks to the following reviewers:
+Frédéric Besson, Lasse Blaauwbroek, Ali Caglayan, Cyril Cohen, Andrej
+Dudenhefner, Andres Erbsen, Jim Fehrle, Emilio Jesús Gallego Arias,
+Gaëtan Gilbert, Jason Gross, Hugo Herbelin, Ralf Jung, Jan-Oliver
+Kaiser, Chantal Keller, Olivier Laurent, Rodolphe Lepigre, Yishuai Li,
+Ralph Matthes, Guillaume Melquiond, Pierre-Marie Pédrot, Karl
+Palmskog, Clément Pit-Claudel, Pierre Rousselin, Pierre Roux, Michael
+Soegtrop, soukouki, Matthieu Sozeau, Nicolas Tabareau, Enrico Tassi,
```

Comment:
@soukouki you're welcome to provide a full name if you prefer (before next Wednesday, sorry for the short notice)